addons/web/static/src/webclient/user_menu/user_menu_items.js

```diff
@@ -53,10 +57,11 @@ export function preferencesItem(env) {
     type: "item",
     description: env._t("Preferences"),
     callback: async function () {
-      const actionDescription = await env.services.orm.call("res.users", "action_get");
+      const actionDescription = await env.services.model("res.users").call("action_get");
```

Comment:
/!\

Reply:
it's a revert. it will be reintroduced soon